cli/helm.go

```diff
@@ -15,32 +15,32 @@ import (
 // and should not be trusted.
 var version = "0.1.0"
 
-const globalUsage = `The Kubernetes package manager
+const globalUsage = `Helm Classic - The original Kubernetes package manager
```

Comment:
I'm going to make the same change to the docs as well-- keeping that in _this_ PR so that this one PR enforced consistency in the tagline.